### PR TITLE
fix(SRV): update functions trigger page and concepts MTA-3644

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: serverless/functions/concepts.mdx [hunk 1]
```diff
@@ -20,6 +20,14 @@ Cold start is the time a function Instance takes to handle a request when it is
 
 <Concept opened>
 
+## CRON trigger
+
+A CRON trigger is a mechanism used to automatically invoke a Serverless Function at a specific time on a recurring schedule. It works similar to a traditional Linux [cron job](https://en.wikipedia.org/wiki/Cron), using the `* * * * *` format.
```
Comment (RoRoJ) [inline serverless/functions/concepts.mdx:25]:
```suggestion
A CRON trigger is a mechanism used to automatically invoke a Serverless Function at a specific time on a recurring schedule. It works similarly to a traditional Linux [cron job](https://en.wikipedia.org/wiki/Cron), using the `* * * * *` format.
```

File: serverless/functions/how-to/add-trigger-to-a-function.mdx [hunk 2]
```diff
@@ -14,12 +14,7 @@ categories:
   - functions
 ---
 
-This page shows you how to configure triggers for your functions. A trigger is an event that activates or invokes a Serverless Function.
-In a serverless architecture, a function is not running constantly, but is rather triggered only when there is an event.
-A trigger is a mechanism that connects the function to an event source and enables the function to execute automatically in response to specific events.
-The trigger could be anything from an HTTP request, a message from a queue or stream, a CRON schedule etc.
-
-The trigger is configured during function deployment, and it determines how and when the function will execute.
+This page shows you how to configure triggers for your functions.Triggers are configured during function deployment, and determine how and when the function will execute.
```
Comment (RoRoJ) [inline serverless/functions/how-to/add-trigger-to-a-function.mdx:17]:
```suggestion
This page shows you how to configure triggers for your functions. Triggers are configured during function deployment, and determine how and when the function will execute.
```

File: serverless/functions/concepts.mdx [hunk 3]
```diff
@@ -132,6 +140,17 @@ Serverless Functions simplify deploying applications to the Cloud. You are only
 
 <Concept>
 
+## SQS trigger
+
+An SQS (Simple Queue Service) trigger is a mechanism that connects a function to an [SQS queue](/serverless/messaging/concepts/#sqs) and invokes the function automatically whenever a message is added to the queue.
```
Comment (nerda-codes) [inline serverless/functions/concepts.mdx:145]: Should the documentation link be #sqs-queue at the end or is this one correct?

Comment (SamyOubouaziz) [reply serverless/functions/concepts.mdx:145]: SQS queue does not exist as a concept. Applied the link to only "SQS" rather than "SQS queue"

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**